core/scripts/Voting.js

```diff
@@ -127,7 +171,10 @@ async function runVoting() {
 }
 
 run = async function(callback) {
+  var request = { identifier: { first: "BTC", second: "USD" }, time: "1560762000" };
```

Comment:
Might be better to delete these from here, until we integrate everything together.